public/src/components/teacher/CourseGroup.tsx

```diff
@@ -229,6 +236,60 @@ export class CourseGroup extends React.Component<ICourseGroupProps, ICourseGroup
         });
     }
 
+    private generateGroupButtons(group: Group): ILink[] {
+        const links = [];
+        switch (group.getStatus()) {
+            case Group.GroupStatus.PENDING:
+                links.push({
+                    name: "Approve",
+                    extra: "primary",
+                    uri: "approve",
+                    description: "list-btn",
+                }, {
+                    name: "Edit",
+                    extra: "primary",
+                    uri: "edit",
```

Comment:
Shouldn't this also have a description?

Reply:
None of these needs a description, as we apply the style to the whole button group, so I've removed it for all list buttons.